lib/src/tar/import.rs

```diff
@@ -606,10 +603,10 @@ pub async fn import_tar(
     let options = options.unwrap_or_default();
     let src = ReadBridge::new(src);
     let repo = repo.clone();
-    let import = tokio::task::spawn_blocking(move || {
+    let import = crate::tokio_util::spawn_blocking_cancellable(move |cancellable| {
```

Comment:
Minor: a `use crate::tokio_util` in this module will make this lighter to read.

Reply:
Hmm, it's the only use though?